### PR TITLE
Hardware page

### PR DESCRIPTION
**Pull request by dmatis2**
Implementation of Hardware page

- content added
- images added

To Do:
- resize several images

### CODE REVIEW

File: content/hk20/hardware.md [hunk 1]
```diff
@@ -16,14 +16,99 @@ hardware:
...
+      content: |
+        An open-source physical computing platform based on a simple microcontroller board, and a development environment for writing software for the board.
+
+        Included:
+        - USB 2.0 Cable
+      image: images/hardware/arduino.png
+
+    - title: Base Shield (5x)
+      content: |
+        Grove module connector.
```
Comment (padr31) [inline content/hk20/hardware.md:34]: tu chyba included

File: content/hk20/hardware.md [hunk 2]
```diff
@@ -16,14 +16,99 @@ hardware:
...
+        Included:
+          - Charger
+          - Additional Straps
+
+      image: images/hardware/fit-bit.png
 ---
 
 {{< section class="pb-0" >}}
-*Our friends from [Major League Hacking](https://mlh.io/) will bring amazing state-of-the-art hardware that you can borrow in case you are participating on-site, to make your vision come true!*
+_Our friends from [Major League Hacking](https://mlh.io/) will bring amazing state-of-the-art hardware that you can borrow in case you are participating on-site, to make your vision come true!_
```
Comment (padr31) [inline content/hk20/hardware.md:111]: Tu by som este pridal nazov podstranky ako je v lectures

File: content/hk20/hardware.md (hunk 1 above)
Comment (dmatis2) [reply content/hk20/hardware.md:34]: Ani v povodnej stranke k tomuto nieje

**Comment by jjurm** [comment]
Please don't change fill-image to fit-image globally. Fill-image is to ensure all images have same aspect ratio when required.
Changing this to fit-image breaks, for example, the Lectures page: "Grafika v hrách" image has wrong aspect ratio.

Alternative: add a parameter for each `serviceItem` to provide custom format for the image. Something similar is done in the `sideimg`, where each image's size can be overridden locally.

**Comment by dmatis2** [comment]
> Please don't change fill-image to fit-image globally. Fill-image is to ensure all images have same aspect ratio when required.
> Changing this to fit-image breaks, for example, the Lectures page: "Grafika v hrách" image has wrong aspect ratio.
> 
> Alternative: add a parameter for each `serviceItem` to provide custom format for the image. Something similar is done in the `sideimg`, where each image's size can be overridden locally.

Can I create a new shortcode, like `fitImage` or so. What do you think @jjurm

**Comment by padr31** [comment]
> Please don't change fill-image to fit-image globally. Fill-image is to ensure all images have same aspect ratio when required.
> Changing this to fit-image breaks, for example, the Lectures page: "Grafika v hrách" image has wrong aspect ratio.
> 
> Alternative: add a parameter for each `serviceItem` to provide custom format for the image. Something similar is done in the `sideimg`, where each image's size can be overridden locally.

I think the change was not changing the fill-image. The client-image was used instead that already has fit image, and then it was renamed so that it has a better name

**Comment by dmatis2** [comment]
Please look at it now @jjurm @padr31

**Pull request merged**